src/components/users/viewCohorts/viewCohorts.jsx

```diff
@@ -1,10 +1,45 @@
-import React from "react"
-import { useParams } from "react-router-dom"
-import "./viewCohort.css"
-import Header from "../../Header/Header.jsx"
+import React from "react";
+import { useState, useEffect } from "react";
+import { useParams } from "react-router-dom";
+import "./viewCohort.css";
+import Header from "../../Header/Header.jsx";
+import client from "../../../utils/client";
 
 export default function ViewCohort() {
-  const { id } = useParams()
+  const { id } = useParams();
+  const [noCohort, setNoCohort] = useState([]);
+
+  useEffect(() => {
+    client
+      .get(`/user`)
+      .then((res) => {
+        const noCo = res.data.data.users.filter(
+          (user) => user.role === "STUDENT" && user.cohort_id === null
+        );
+        setNoCohort(noCo);
+      })
+      .catch((err) => console.log(err.response));
+  }, []);
+
+  const addStudent = (event) => {
+    const token = localStorage.getItem(process.env.REACT_APP_USER_TOKEN);
+    const data = { cohort_id: id, id: parseInt(event.target.value) };
+    let headers = { "Content-type": "application/json" };
+    headers["authorization"] = `Bearer ${token}`;
+
+    const options = {
+      method: "PATCH",
+      headers: headers,
+      body: JSON.stringify(data),
+    };
+
+    fetch(
```

Comment:
Rather than use fetch direcrtly, can you use one of the methods on the `client` object? That will add the token and headers for you. See `utils/client.js`.

Reply:
there isnt a patch request in there, but we could make one?

Reply:
Definitely 👍